src/cmd/services/m3query/config/config.go

```diff
@@ -94,6 +94,9 @@ type Configuration struct {
 	// ListenAddress is the server listen address.
 	ListenAddress *listenaddress.Configuration `yaml:"listenAddress" validate:"nonzero"`
 
+	// The host and port on which to listen for debug endpoints.
+	DebugListenAddress string `yaml:"debugListenAddress"`
```

Comment:
Might be a good idea to get into the habit of editing [annotated config](https://sourcegraph.com/github.com/m3db/m3@master/-/blob/docs/query_engine/config/annotated_config.yaml#L50:21) when we add config settings (think it's quite behind unfortunately)

Reply:
Yeah I'm working on restructuring the docs so lemme jot this down.